integration/e2emimir/services.go

```diff
@@ -23,13 +23,21 @@ const (
 func GetDefaultImage() string {
 	// Get the mimir image from the MIMIR_IMAGE env variable,
 	// falling back to grafana/mimir:latest"
-	if os.Getenv("MIMIR_IMAGE") != "" {
-		return os.Getenv("MIMIR_IMAGE")
+	if img := os.Getenv("MIMIR_IMAGE"); img != "" {
+		return img
 	}
 
 	return "grafana/mimir:latest"
 }
 
+func GetMimirtoolImage() string {
+	if img := os.Getenv("MIMIR_IMAGE"); img != "" {
```

Comment:
Shouldn't we use a different env var, like `MIMIRTOOL_IMAGE`?

Reply:
💯 right! Thanks for spotting it!